.github/workflows/ci.yml

```diff
@@ -32,6 +32,7 @@ jobs:
           -D clippy::expect_used
           -D clippy::panic
           -D clippy::unwrap_used
+          -A clippy::uninlined_format_args
```

Comment:
Why disable this lint vs fixing the inlines?

Reply:
As there were many required changes to fix the lint, I intended to leave a comment and make those fixes in a separate PR, but I found out `uninlined_format_args` became pedantic again in Rust [1.67.1](https://blog.rust-lang.org/2023/02/09/Rust-1.67.1.html#whats-in-1671-stable) released last week, so we no longer need this. The latest GitHub Actions ubuntu-latest image [should have this Rust version now](https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2204-Readme.md#rust-tools).